versions.tf

```diff
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 0.13.1"
+  required_version = ">= 1.3.0"
```

Comment:
This is a breaking change and we do not want to push users to the bleeding edge as it will reduce adoption